.gitignore

```diff
@@ -1,5 +1,7 @@
 # Cache, temp and personal files
 
+.env
+.idea
```

Comment:
What do you mean? You shouldn't commit .idea stuff or your .env file?

Reply:
I was talking about the `.idea` file

Reply:
you wouldn't want to commit the .idea folder, so why wouldn't you include it in the .gitignore?

Reply:
Best practices is to add it on global gitignore. We don't overload the the PrestaShop .gitignore with each IDEA / custom local configurations.

Reply:
The `.env` makes sense since this is a Symfony project, however `.idea` is only related to the IDEA you are using, so it shouldn't be in the global shred ignore or we'll start adding specific rules for each contributor's environment (IDE configs, OS specific hidden files, ...).

But you can configure your own local environment to ignore the `.idea` folders on all your cloned repository, which is why this should be a local configuration:

```sh
git config --global core.excludesfile ~/.gitignore
```

Reply:
You can read https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/ for example ;-)